protocol_tests/aws-restjson/tests/functional/aws_rest_json_1_1.spec.ts

```diff
@@ -191,7 +191,7 @@ it("RestJsonAllQueryStringTypes:Request", async () => {
 
     queryStringList: ["a", "b", "c"],
 
-    queryStringSet: new Set(["a", "b", "c"]),
+    queryStringSet: ["a", "b", "c"],
```

Comment:
> Does it make sense to have separate Set/List in this case?

Ideally, the output needs to be verified with input instead of doing them separately. Created an issue for it at https://github.com/aws/aws-sdk-js-v3/issues/1171

> Can we simply remove queryStringSet altogether?

We can't as the protocol_tests are codegen from https://github.com/awslabs/smithy/blob/34d51e3f7f9c351ed0867a816e082ba3ac56cc4c/smithy-aws-protocol-tests/model/restXml/http-query.smithy#L103